tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -378,6 +379,18 @@ const getDisplayedContactsNames = async () => {
   return contacts;
 };
 
+const getErrorLog = async () => {
```

Comment:
Hi @jonathanbataire, Have you had the chance to look at @lorerod feedback?

Reply:
sorry totally missed this 
@lorerod the selectors for the errorlog are different in both. thats the only diff
`const errorLogSelector = () => $('.item-content error-log');` in contacts  vs `const errorLog = () => $('.page error-log');` in analytics
im pretty divided on how to go about this. isnt the difference significant enough to separate the 2? idk
lmk what you think.

Reply:
Hi @jonathanbataire @latin-panda, what do you think about moving the method to the `common.wdio.page.js` and making it receive the `errorLog` selector as a parameter?

Reply:
Hi all,
I don't know if the error text is the same for contacts, analytics, and tasks, but if it is, we can use a selector by text, and it will be the same method for all three places.

@jonathanbataire look [this example](https://github.com/medic/cht-core/blob/master/tests/page-objects/upgrade/upgrade.wdio.page.js#L4-L6) of the upgrade page object file, in in which we use the selector by text 🙂 

Let me know if I can help..

Reply:
Hello @tatilepizs @lorerod this has been updated getErrorLog moved to `commompage`

Reply:
That is great, thank you @jonathanbataire for making the change and updating all the instances ⭐ this really help us to keep our code clean and organized 👍